### PR TITLE
Pull request for master

### PR DESCRIPTION
**Pull request by tonellotto**
## syntax sugar

### CODE REVIEW

File: pyterrier/__init__.py [hunk 1]
```diff
@@ -15,17 +11,22 @@
...
     Function necessary to be called before Terrier classes and methods can be used.
     Loads the Terrier.jar file and imports classes. Also finds the correct version of Terrier to download if no version is specified.
 
     Args:
         version(str): Which version of Terrier to download. Default=None.
             If None, find the newest Terrier version in maven and download it.
-        mem: Maximum memory allocated for java heap in MB. Default=4096.
-        packages: Extra .jar files to load.
+        mem(str): Maximum memory allocated for java heap in MB. Default=4096.
+        packages(list(str)): Extra .jar files to load. Default=[].
```
Comment (cmacdonald) [inline pyterrier/__init__.py:23]: this should refer to Maven package coordinates, not jar files.

File: pyterrier/__init__.py [hunk 2]
```diff
@@ -15,17 +11,22 @@
...
     """
     Function necessary to be called before Terrier classes and methods can be used.
     Loads the Terrier.jar file and imports classes. Also finds the correct version of Terrier to download if no version is specified.
 
     Args:
         version(str): Which version of Terrier to download. Default=None.
             If None, find the newest Terrier version in maven and download it.
-        mem: Maximum memory allocated for java heap in MB. Default=4096.
-        packages: Extra .jar files to load.
+        mem(str): Maximum memory allocated for java heap in MB. Default=4096.
```
Comment (cmacdonald) [inline pyterrier/__init__.py:22]: Please, lets leave None as default. I think JVM will use 25% of system RAM by default, which is less brittle than a hard-coded number.

File: tests/test_trec_indexer.py [hunk 3]
```diff
@@ -26,6 +26,7 @@ def tearDown(self):
 
     def test_TREC_indexing(self):
         indexer = pt.TRECCollectionIndexer(self.test_dir)
+        pt.redirect_stdouterr()
```
Comment (cmacdonald) [inline tests/test_trec_indexer.py:29]: Was this intended? It should be default now

File: pyterrier/__init__.py (hunk 1 above)
Comment (tonellotto) [reply pyterrier/__init__.py:23]: Ok. I just left the message as written.

File: pyterrier/__init__.py (hunk 2 above)
Comment (tonellotto) [reply pyterrier/__init__.py:22]: Ok, but we should clearly state that in the doc: 4 GB is not the default on my server, according to the 25% rule.

File: pyterrier/__init__.py (hunk 1 above)
Comment (cmacdonald) [reply pyterrier/__init__.py:23]: While we are here, we should improve things!

File: tests/test_trec_indexer.py (hunk 3 above)
Comment (tonellotto) [reply tests/test_trec_indexer.py:29]: yep. debugging leftover.

**Comment by cmacdonald** [comment]
Thanks. Once tests pass, I will merge.

**Pull request merged**